src/Helpers/LayoutHelper.php

```diff
@@ -194,11 +197,10 @@ private static function makeSidebarClasses()
 
         // Add classes related to the "sidebar_mini" configuration.
 
-        $sidebarMiniCfg = config('adminlte.sidebar_mini', 'lg');
```

Comment:
This should not be deleted, it was part of a recent contribution... I think you need update your master branch to the latest state and then rebase your AdminLTEIFrame branch.